openbb_platform/core/openbb_core/app/logs/logging_service.py

```diff
@@ -217,42 +217,44 @@ def log(
         )
         self._handlers_manager.update_handlers(self._logging_settings)
 
-        if "login" in route:
-            self._log_startup(route, custom_headers)
-        else:
-
-            # Remove CommandContext if any
-            kwargs.pop("cc", None)
-
-            # Get provider for posthog logs
-            passed_model = kwargs.get("provider_choices", DummyProvider())
-            provider = (
-                passed_model.provider
-                if hasattr(passed_model, "provider")
-                else "not_passed_to_kwargs"
-            )
-
-            # Truncate kwargs if too long
-            kwargs = {k: str(v)[:100] for k, v in kwargs.items()}
-            # Get execution info
-            error = None if all(i is None for i in exec_info) else str(exec_info[1])
-
-            # Construct message
-            message_label = "ERROR" if error else "CMD"
-            log_message = json.dumps(
-                {
-                    "route": route,
-                    "input": kwargs,
-                    "error": error,
-                    "provider": provider,
-                    "custom_headers": custom_headers,
-                },
-                default=to_jsonable_python,
-            )
-            log_message = f"{message_label}: {log_message}"
-            log_level = self._logger.error if error else self._logger.info
-            log_level(
-                log_message,
-                extra={"func_name_override": func.__name__},
-                exc_info=exec_info,
-            )
+        if not self._logging_settings.logging_suppress:
+
+            if "login" in route:
+                self._log_startup(route, custom_headers)
+            else:
+
+                # Remove CommandContext if any
+                kwargs.pop("cc", None)
+
+                # Get provider for posthog logs
+                passed_model = kwargs.get("provider_choices", DummyProvider())
+                provider = (
+                    passed_model.provider
+                    if hasattr(passed_model, "provider")
+                    else "not_passed_to_kwargs"
+                )
+
+                # Truncate kwargs if too long
+                kwargs = {k: str(v)[:100] for k, v in kwargs.items()}
```

Comment:
since we are in here -- can we do future me a solid and make this a larger clip.  maybe like :300

Reply:
300 it is.